### DISCUSSION TITLE
Is 3.8.1 ready for usage?

### DISCUSSION THREAD

**Discussion by dchirita** [CocoaLumberjack/CocoaLumberjack#1369] [General]
Hello,

I noticed that 3.8.1 is missing from releases. The tag can be consumed but no release is created yet.

Was it just missed or there is something extra needed before creating a release?

Thanks!

Daniel

**Comment by ffried** [comment]
@dchirita I'm not sure I follow. I see [release 3.8.1](https://github.com/CocoaLumberjack/CocoaLumberjack/releases/tag/3.8.1) both in the righthand releases column, as well as marked as 'latest release' in the releases page.

<img width="321" alt="image" src="https://github.com/CocoaLumberjack/CocoaLumberjack/assets/1689782/5c58920d-6103-4796-adb1-7b2d7fe83476">
<img width="1211" alt="image" src="https://github.com/CocoaLumberjack/CocoaLumberjack/assets/1689782/8b0e1ff9-b27f-4a32-b830-e6685be56821">

**Reply by dchirita** [reply]
Hey @ffried 

You can see the tag here:
https://github.com/CocoaLumberjack/CocoaLumberjack/tags

but you can not see the release here:
https://github.com/CocoaLumberjack/CocoaLumberjack/releases

Am I missing something here?

**Reply by ffried** [reply]
<img width="1620" alt="image" src="https://github.com/CocoaLumberjack/CocoaLumberjack/assets/1689782/58324cfc-3971-4bbf-9c90-8b5747cc4179">

**Reply by ffried** [reply]
Maybe there's a (very) outdated cache somewhere?

**Comment by dchirita** [comment]
@ffried Thanks for your reply! This is what I get.

Tried a different browser, tried deleting the cache, and tried incognito, cURL. Nothing works.

Could you try in a different browser too? thanks in advance.

!! In any case, I understand that this is a good-to-go release which is what I needed. Thanks!

<img width="2041" alt="Screenshot 2023-09-05 at 16 12 29" src="https://github.com/CocoaLumberjack/CocoaLumberjack/assets/3481081/5a1c4c16-65e1-4845-a8cb-6ee65015a246">

**Reply by ffried** [reply]
@dchirita Very weird. I just edited the release and saved it again. Now it seems to show up consistently in `curl` for me. Thanks for informing us about this!

**Reply by dchirita** [reply]
I confirm it looks ok now. Thank you! All the best!
